### PR TITLE
Remove Maestro

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

**Comment by dougbu** [comment]
I don't think PR validation will work unless you either stop the pipeline or remove the Maestro/ mentions from azure-pipelines.yml.

as a smaller detail, suggest removing the https://github.com/mmitche/versions/tree/remove-maestro?tab=readme-ov-file#maestro section in the readme

**Comment by mmitche** [comment]
Removed the pipeline and YAML and updated the readme

File: README.md [hunk 1]
```diff
@@ -1,76 +1,6 @@
 # .NET Versions
-This repo contains information about the various component versions that ship with .NET Core. 
 
-## build-info
```
Comment (dougbu) [inline README.md:4]: The new info about the 3 build-info/ subfolders is great and I agree `Maestro` had to go.

Suggest restoring this information or moving it into a new build-info/ readme. Losing about the only way to navigate that tree other than blindly searching around sub-folders. Admitedly some of the terminology is Maestro-centric but most of the names appear as file and folder names w/in build-info/.

Comment (mmitche) [reply README.md:4]: I'll move the info into dotnet and label it as legacy information. I was worried that the existence of it implies that any of it is still active. We haven't done a build that uses this infra in many many years.

**Review by dougbu — APPROVED** [review]

**Pull request merged**